README.rst

```diff
@@ -46,11 +46,11 @@ __ `Ubuntu package`_
 Docker
 ------
 
-A docker package exists for this project: 
+A docker package exists for this project::
```

Comment:
Why the double colons here and in other places?

Reply:
@auouymous 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Reply:
My first time using RST - I saw this used many times in this README,  isn't this how one is supposed to do a codeblock?

Reply:
Oops, I thought it was markdown.